doc/whats_new.rst

```diff
@@ -29,6 +29,7 @@ Bug
 API
 ~~~
 
+- Add support for non maxfiltered .fif files, by `Maximilien Chaumon`_ (`#171 <https://github.com/mne-tools/mne-bids/pull/171>`_)
```

Comment:
Your name will not render with a link unless you also provide a website down below like the others.

Reply:
fine with me. Thanks

Reply:
I meant that you need to add the weblink at the bottom. You can give your github weblink. Or alternatively, remove the markup for links.